modules/appeals_api/spec/requests/v1/notice_of_disagreements/evidence_submissions_controller_spec.rb

```diff
@@ -26,4 +26,25 @@
       expect(body).to match_array(serialized)
     end
   end
+
+  describe '#create' do
+    let(:fixtures_path) { '/modules/appeals_api/spec/fixtures' }
+    let(:uploaded_file) { Rack::Test::UploadedFile }
+    let(:valid_upload_params) do
+      { document: uploaded_file.new("#{::Rails.root}#{fixtures_path}/expected_10182_extra.pdf"), uuid: '1234' }
```

Comment:
Only thing I'd say is to pull this root/fixtures_path out, but looks good otherwise!

Reply:
Ugh. I was doing this to avoid getting pulled over by rubocop. I can't stand forcing things on a million different lines if I don't have to, so I went with this. I still prefer this to multiple lines, but I could be persuaded...

Reply:
Feel free to add some extra helper functions into our spec/support files that make getting the fixture path easier, if they don't already exist.